docs/reference-guides/block-api/block-edit-save.md

```diff
@@ -183,9 +183,34 @@ save: ( { attributes } ) => {
 ```
 
 
-
 When saving your block, you want to save the attributes in the same format specified by the attribute source definition. If no attribute source is specified, the attribute will be saved to the block's comment delimiter. See the [Block Attributes documentation](/docs/reference-guides/block-api/block-attributes.md) for more details.
 
+### innerBlocks
+
+There is a second property in the props passed to the `save` function, `innerBlocks`. This property is typically used for internal operations, and there are very few scenarios where you would need to use it.
+
+`innerBlocks`, when initialized, is an array containing object representations of nested blocks. In those rare cases where you might use this property,
+it can help you adjust how a block is rendered. For example, you could render a block differently based on the number of nested blocks or if a specific block type is present..
+
+
+```jsx
+save: ( { attributes, innerBlocks } ) => {
```

Comment:
It looks like it's a convenient alternative to another way to check inner blocks:

https://github.com/WordPress/gutenberg/blob/c9790844331f8a35da15d415f5775a80765e922b/packages/block-library/src/gallery/save.js#L24-L29

I'm not entirely sure what the difference exactly is. However, I wanted to document it for completeness.